### ISSUE TITLE
Enhance swiftenv to support ARM based Linux

### ISSUE THREAD

**Issue by segabor** [kylef/swiftenv#175]
I started using Swift 5.2 on my Raspberry Pi 4 running Ubuntu Server. `swiftenv` is a great tool to install and manage Swift versions and it would be super cool to have it working on Linux / ARM (aarch64 architecture).
Of course scripts do work but `swiftenv-install` wants to install 'default' Intel based Swift package.

Do you plan enhancing `swiftenv` to properly detect and deliver OS architecture to `swiftenv-install` so it can download the right package on ARM based Linux?

**Comment by kylef** [comment]
The Swift project only appear to offer x86_64 binaries (I don't see any aarch64 at https://swift.org/download/#releases), happy to support it if that changes.

**Comment by segabor** [comment]
Indeed, no official binaries yet. However @futurejones is doing a great job porting and maintaining Swift for various ARM based Linux distributions. Pre-built packages are available here: https://packagecloud.io/swift-arm/release
I’m not sure if they are packaged the same way as the official ones by Apple.
